lib/features/modeling/behavior/RemoveParticipantBehavior.js

```diff
@@ -41,8 +49,70 @@ export default function RemoveParticipantBehavior(eventBus, modeling) {
     }
   }, true);
 
+  this.postExecute('shape.replace', function(context) {
```

Comment:
Maybe add a comment what this part of the behavior is doing. Alternatively: Move the reconnect message flow behavior to an entirely different component. It is fairly standalone and I could imagine that I'll never again find where we implemented it in this file :see_no_evil:

Reply:
Makes sense. There are limitations to the method of just adding new behaviors to an existing behavior that has an affected element in the file name. 🤡 I will move this to a separate behavior.